client/client-lib/src/mint/mod.rs

```diff
@@ -155,6 +153,110 @@ impl MintClient {
         self.context.db.begin_transaction(ModuleRegistry::default())
     }
 
+    /// Adds the final amounts of `change` to the tx before submitting it
+    /// Allows for multiple `change` outputs
+    pub async fn finalize_change(&self, tx: &mut Transaction, change: Vec<Amount>) {
```

Comment:
Should probably not be `pub` since multiple on the same tx are unsafe (tx id changes, DB entries become invalid).